ambry-server/src/test/java/com.github.ambry.server/AmbryRequestsTest.java

```diff
@@ -528,6 +538,53 @@ public void stopBlobStoreFailureTest() throws InterruptedException, IOException
     assertTrue("Response not of type AdminResponse", response instanceof AdminResponse);
     storageManager.exceptionToThrowOnShuttingDownBlobStore = null;
   }
+
+  /**
+   * Tests blobIds can be converted as expected and works correctly with GetRequest.
+   * If all blobIds can be converted correctly, no error is expected.
+   * If any blobId can't be converted correctly, Blob_Not_Found is expected.
+   * @throws InterruptedException
+   * @throws IOException
+   */
+  @Test
+  public void listOfOriginalStoreKeysGetTest() throws Exception {
```

Comment:
Is there an explicit deletion test?  I'm not sure if I'm seeing it

Reply:
There isn't - only the integration tests test that unfortunately (this class didn't have any tests until a few months ago)

Reply:
It's covered by existing test `controlRequestSuccessTest`. Without the conversion, delete fails. 

`GetRequest` may request multiple keys, so `listOfOriginalStoreKeysGetTest ` was added for enhancement.

Reply:
Ok cool, so all new code has coverage (when run through "Run * with coverage")?

Reply:
Yes. New changes covered.